### ISSUE TITLE
Rule proposal: extend prefer-modern-dom-apis with insertBefore => prepend

### ISSUE THREAD

**Issue by clshortfuse** [sindresorhus/eslint-plugin-unicorn#1715]
### Description

Any usage of `element.insertBefore(child, element.firstChild)` can be converted to `element.prepend(child)`. 

(Note, this isn't safe with `element.firstChildElement`)

### Fail

```js
element.insertBefore(child, element.firstChild);
```

### Pass

```js
element.prepend(child)
```

**Comment by silverwind** [comment]
Can [`element.before`](https://developer.mozilla.org/en-US/docs/Web/API/Element/before) also be covered?
